src/main/java/seedu/address/model/person/Deadline.java

```diff
@@ -60,6 +81,15 @@ public int hashCode() {
         return value.hashCode();
     }
 
+    @Override
+    public int compareTo(Deadline other) throws ClassCastException {
```

Comment:
I honestly don't know if the way I handled the exception is ideal, but the reason I throw the ClassCastException instead of directly throwing ParseException is because compareTo is an overriden method, so I can only throw exceptions that the original method throws.

Reply:
I see! It looks like we shouldn't encounter this exception given how we've restricted its use, so this way is fine. 😄